### ISSUE TITLE
A few typos

### ISSUE THREAD

**Issue by danuSHekar** [cs231n/cs231n.github.io#236]
Under section "Layer Sizing Patterns" of the "Convolutional Neural Networks: Architectures, Convolution / Pooling Layers" chapter in Module 2, there seems to be an incomplete sentence: "Another slightly less common setting is to use 3x3 receptive fields with a stride of 2, but this makes."
Under section "Mini-batch gradient descent" of the Optimisation chapter in Module 1, there is an incorrect sentence: "the training data can have on order of millions of examples" instead of "the training data can have an order of millions of examples".
